core/server/models/post.js

```diff
@@ -260,6 +260,18 @@ Post = ghostBookshelf.Model.extend({
             this.set('tags', tagsToSave);
         }
 
+        model.related('tags').on('detaching', function onDetached(collection, tag) {
+            model.related('tags').on('detached', function onDetached(detachedCollection, response, options) {
+                tag.emitChange('edited', options);
+            });
+        });
+
+        model.related('tags').on('attaching', function onDetached(collection, tags) {
+            model.related('tags').on('attached', function onDetached(detachedCollection, response, options) {
+                tags.forEach(tag => tag.emitChange('edited', options));
```

Comment:
Have you thought about triggering a proper "attached"/"detached" event?
e.g `tag.attached`

By this we don't need to abuse `_onResourceUpdated`. Because the logic is now a little bit hard to understand (we now remove and add resources in this function, but it's an update resource event).

Reply:
Furthermore: if we re-use `tag.edited`, we will trigger an edited event although the tag was not edited
this can cause problems and confusion if you listen on this event elsewhere in Ghost.